pkg/blobstore/configuration/create_blob_access.go

```diff
@@ -261,12 +261,41 @@ func createBlobAccess(configuration *pb.BlobAccessConfiguration, options *blobAc
 				backend.Redis.ReplicationCount,
```

Comment:
It looks like we hardcode these values for the clustered Redis solution. Could you please also use those options there? Be sure to move all of the duration conversion/parsing above the `switch` statement, so that it can be reused in both branches.